openstack/tools/charmed_openstack_functest_runner.sh

```diff
@@ -76,6 +76,15 @@ do
     shift
 done
 
+TOOLS_PATH=$(realpath $(dirname $0))/func_test_tools
+CHARM_PATH=$(pwd)
+
+# Get commit we are running tests against.
+COMMIT_ID=$(git -C $CHARM_PATH rev-parse --short HEAD)
+CHARM_NAME=$(egrep -o "^name: .+" metadata.yaml| cut -f 2 -d ' ')
```

Comment:
If you don't mind installing `yq`:
`CHARM_NAME=$(yq '.name' metadata.yaml)`

Or just `awk`:
`CHARM_NAME=$(awk '/name: .+/{print $2}' metadata.yaml)`

(one less pipe)

Reply:
done.